Device/ARMCA53/Config/mem_ARMCA53.h

```diff
@@ -0,0 +1,100 @@
+/**************************************************************************//**
+ * @file     mem_ARMCA53.h
+ * @brief    Memory base and size definitions (used in scatter file)
+ * @version  V1.0.0
+ * @date     31. March 2024
+ *
+ * @note
+ *
+ ******************************************************************************/
+/*
+ * Copyright (c) 2009-2019 Arm Limited. All rights reserved.
```

Comment:
Is the year in the copyright header correct?
I think it should be only 'Copyright (c) 2024 Arm Limited.'

Please check also the other copyright headers.

Reply:
All copyrights should now be set to 2024